dapp/src/utils/utils.js

```diff
@@ -33,3 +37,52 @@ export function mergeDeep(target, ...sources) {
 
   return mergeDeep(target, ...sources)
 }
+
+/**
+ * Get stable coin logs
+ */
+
+export const getStableCoinLogs = async (transactions) => {
+  const data = {
+    usdt: 0,
+    dai: 0,
+    usdc: 0,
+    ousd: 0,
+  }
+  const stableCoins = [
+    {
+      address: addresses.mainnet.USDT.toLowerCase(),
```

Comment:
Hardcoding those addresses to mainnet will cause this function to fail on any other network than mainnet. For example on local hardhat network we use for development.

How about leveraging the addresses already loaded in the react state by this method: https://github.com/OriginProtocol/origin-dollar/blob/dc9ec97fadb90b409be0f7749be97a9b149d1855/dapp/src/utils/contracts.js#L20

Reply:
@franckc to use this variables we need to put these variables inside store. but I don't see it's implemented yet.

Reply:
@jupiteruwdev for sharing a global state we are using `pullstate` instead of say Redux. Here is where the contract data gets stored into the global state: https://github.com/OriginProtocol/origin-dollar/blob/dc9ec97fadb90b409be0f7749be97a9b149d1855/dapp/src/utils/contracts.js#L351-L353

Reply:
i see